resources/entrypoint.sh

```diff
@@ -23,5 +23,12 @@ if [ "$EXIT" -eq 1 ]; then
     exit 1
 fi
 
+# We need a host to send our stats. I happen to know that our collector is
```

Comment:
We should support overriding STATSD_HOST in the container's environment. Something like this will work:

```
CONTAINER_HOST_IP=$(ip route show | awk '/default/ {print $3}')
export STATSD_HOST=${STATSD_HOST:-$CONTAINER_HOST_IP}
```
That syntax will default STATSD_HOST to the value of the STATSD_HOST environment variable, if it's set, otherwise use the CONTAINER_HOST_IP set above.

Reply:
Done